initial version Order DataMiner credits (#2604)
SkylineCommunications/dataminer-docs: user-guide/Getting_started/DaaS/Order_DataMiner_credits.md

---
uid: Order_DataMiner_credits
---

# Order DataMiner credits

> 1. Go to the [Azure Marketplace](https://azuremarketplace.microsoft.com/) and Search for DataMiner.

Reviewer: ```suggestion
1. Go to the [Azure Marketplace](https://azuremarketplace.microsoft.com/) and search for DataMiner.
```

    ![Azure Marketplace DataMiner credits offer](~/user-guide/images/Azure_Marketplace_Offer.png)

1. Click on *Get It Now* and continue to Azure portal.

> [!WARNING]

> > make sure you have selected the plan Dataminer credits.

Reviewer: ```suggestion
> Make sure you selected the plan DataMiner credits.
```

> 1. Once redirected to the Azure portal click on Subscribe. This will start the process of creating a SaaS subscription resource to be deployed in a resource group.

Reviewer: ```suggestion
1. Once redirected to the Azure portal click on *Subscribe*. This will start the process of creating a SaaS subscription resource to be deployed in a resource group.
```

1. Using your new SaaS subscription click on *Configure account now*. This will redirect you to our marketplace landing page.
    ![Configure account now](~/user-guide/images/Azure_Portal_Configure.png)

> 1. After confirming all you details click on *Purchase credits*. This will activate your subscription and may take a couple minutes.

Reviewer: ```suggestion
1. After confirming all you details click on *Purchase credits*. This will activate your subscription and may take a couple of minutes.
```

OR

```suggestion
1. After confirming all you details click on *Purchase credits*. This will activate your subscription and may take a few minutes.
```

> 1. Once your subscription is activated, Fill in your Organization ID

Reviewer: ```suggestion
1. Once your subscription is activated, fill in your Organization ID
```

>     1. To find your organization ID, Go to [dataminer.services](https://dataminer.services).

Reviewer: ```suggestion
    1. To find your organization ID, go to [dataminer.services](https://dataminer.services).
```

Reviewer: Maybe this step by step guide should be in one of those info boxes?

    1. Log in using your dataminer.services account.

>     1. Select the organization you want to transfer DataMiner credits to.

Reviewer: ```suggestion
    1. Select the organization you want to purchase DataMiner credits for.
```

    1. Go to the webpage URL and copy the last part of the URL after the final slash.
   ![How to retrieve organization ID](~/user-guide/images/Retrieve_Organization_ID.gif)

1. Use the slider to indicate how many credits you wish to purchase for the organization.

> 1. Click on *Purchase credits* and confirm your purchase

Reviewer: ```suggestion
1. Click on *Purchase credits* to confirm your purchase
```

> [!NOTE]

> > If you would like to order more credits you do not have to create a new SaaS subscription. You can simply go to your SaaS subscription in the Azure portal and click on *Open SaaS Account on publisher's site* to be redirected again to our page.

Reviewer: ```suggestion
> If you would like to order more credits, you do not have to create a new SaaS subscription. You can simply go to your SaaS subscription in the Azure portal and click on *Open SaaS Account on publisher's site* to be redirected to our page again.
```

Reviewer: Maybe find a better way to say "our page"? Like purchase/order page or something?

---
uid: Order_DataMiner_credits
---

# Ordering DataMiner credits

To create and maintain a DaaS instance, your [organization](xref:Pricing_Usage_based_service#organization) must have sufficient DataMiner credits. To order DataMiner credits for the first time, follow the procedure below.

> [!NOTE]
> If you have already created a SaaS subscription as detailed below, and you want to order **additional credits**, you do not have to create a new SaaS subscription. Instead, go to your SaaS subscription in the Azure portal, and click *Open SaaS Account on publisher's site* to be redirected to the *Purchase credits* page.

1. Go to the [Azure Marketplace](https://azuremarketplace.microsoft.com/) and search for DataMiner.

   ![Azure Marketplace DataMiner credits offer](~/user-guide/images/Azure_Marketplace_Offer.png)

> 1. Click *Get It Now* and continue to the Azure portal.

Reviewer 2: Does clicking "Get it Now" redirect the user to the Azure portal or do they still need to do something else to continue? Maybe rephrase this so no confusion is possible.

Author: Get it Now will redirect the user to the Azure portal , the user may or may not need to log in to the portal

   > [!IMPORTANT]

>    > Make sure you have selected the plan DataMiner credits.

Reviewer 2: Where should the user select this? It might be better to make this a separate step before the "Click Get it Now" step, and to add a screenshot.

1. Once you have been redirected to the Azure portal, click *Subscribe*.

>    This will start the process of creating a SaaS subscription resource to be deployed in a resource group.

Reviewer 2: This isn't entirely clear to me, but I'm not sure what the best approach would be. If this is essential information, could you make it more clear what you mean with "to be deployed in a resource group"? Or if the user doesn't really need to know this, could you change it to just "This will start the process of creating a SaaS subscription."?

> 1. Using your new SaaS subscription, click *Configure account now*.

Reviewer 2: What do you mean with "Using your new SaaS subscription"? Does the user need to do something specific to select this? If not, it might be better to leave this out and simplify the step to just "1. Click *Configure account now*."

>    This will redirect you to our marketplace landing page.

Reviewer 2: I'd prefer to rephrase "our" marketplace landing page, because this is a bit vague. Is this the "DataMiner marketplace landing page"?

   ![Configure account now](~/user-guide/images/Azure_Portal_Configure.png)

> 1. After confirming all you details, click *Purchase credits*.

Reviewer 2: What do you mean with "confirming all your details"? If this is a separate action the user needs to do, it would be better to make this a separate step before this one.

Author: the initial screen shows the following:
![image](https://github.com/SkylineCommunications/dataminer-docs/assets/146821343/fcf02b3b-eaa8-4943-afae-391de98e5854)
with this step i mean that the user should just look over their personal details (email) and if they are using the correct account they can click *Purchase credits*

---
uid: Order_DataMiner_credits
---

# Ordering DataMiner credits

To create and maintain a DaaS instance, your [organization](xref:Pricing_Usage_based_service#organization) must have sufficient DataMiner credits. To order DataMiner credits for the first time, follow the procedure below.

> [!NOTE]
> If you have already created a SaaS subscription as detailed below, and you want to order **additional credits**, you do not have to create a new SaaS subscription. Instead, go to your SaaS subscription in the Azure portal, and click *Open SaaS Account on publisher's site* to be redirected to the *Purchase credits* page.

1. Go to the [Azure Marketplace](https://azuremarketplace.microsoft.com/) and search for DataMiner.

1. Click *Get It Now* to be redirected to the Azure portal.

   > [!IMPORTANT]

>    > Make sure you have selected the plan *DataMiner credits*.

Reviewer 2: My original feedback still applies here: it would be better to add this as a separate step (with screenshot), before the "Get It Now" step. Otherwise, you risk that a user clicks "Get It Now" before they see that they should have had "DataMiner credits" selected.

1. Once you have been redirected to the Azure portal, click *Subscribe*.

>    This will start the process of creating a SaaS subscription resource to be deployed in a resource group. You will need to provide a resource name for the SaaS resource and assign the resource to a resource group of your choosing (or make a new resource group). Finally click on *Review + subscribe*.

Reviewer 2: These are several steps in one, which something we prefer to avoid because it's not very user-friendly. Each numbered step should correspond to one separate action. The result of a step should be described in an indented line below the step itself when necessary. Could you adjust this so it's clear to the user what they should do, step by step?
